### DISCUSSION TITLE
Dangerous links found in wiki

### DISCUSSION THREAD

**Discussion by huwdp** [lightspark/lightspark#966] [General]
The user here (https://github.com/Iightspark) is editing the wiki with their own potentially dangerous software. Just wanted to let everyone know.

See history of: https://github.com/lightspark/lightspark/wiki/Getting-Lightspark-up-and-running-in-Web-Browser/_history

@dbluelle

**Comment by eiglow** [comment]
Nice work keeping on top of this.

I reported the user. I suggest you to do the same, this is clearly malicious behaviour.

**Comment by huwdp** [comment]
I've done so too and downloaded the files for inspection when I get time.

Thanks to @poudink for reverting the Pale Moon link too.

**Comment by poudink** [comment]
they did it again. should probably enable edit protection for the page, if github has that feature

**Comment by eiglow** [comment]
Looks like their account has been deleted, the reporting of their account was successful.

**Comment by huwdp** [comment]
@dbluelle , the Getting started up and running page has been modified again. I've reverted it back to what it should be.

I'll check if anything malicious was put there tomorrow.

**Reply by masatake** [reply]
The sha256sum reports the same hash value for the file downloaded from https://github.com/lightspark/lightspark/releases/tag/0.8.6.1 and for the file downloaded from the over-rewritten page.
